code/index.html

```diff
@@ -1,22 +1,67 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
 
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
+<head>
+  <meta charset="utf-8" />
+  <title>Oldflån Season Magazine</title>
+  <link rel="stylesheet" href="./style.css" />
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+  <!-- dont forget the meta tag for responsiveness -->
+</head>
 
-    <section class="">
-      <!-- The other news  -->
+<body>
+  <!-- header, logo, navbar -->
+  <header class="logo">
+    <img src="./OlflånLogo.svg" alt="Logo Oldflån">
+    <h1>Oldflån // Season // Magazine</h1>
+  </header>
+  <nav class="menu">
+    <a href="#">Summer</a>
+    <a href="#">Autumn</a>
+    <a href="#">Winter</a>
+    <a href="#">Spring</a>
+  </nav>
+    <!-- Head news -->
+    <section class="head-news-container">
+      <div class="head-news">
+        <h2> New hiking trail between Ansätten and Önrun</h2>
+        <img src="./nöjdfjället2.jpeg" alt="nöjdfjället">
+        <p>Leden upp till fjället Önrun i Åkersjön är lätt att vandra och man kommer högt upp och kan njuta av milsvid
+          utsikt. Önrun sägs ha ”Sveriges vidaste vy”. Toppen ligger på 856 möh.Leden upp till fjället Önrun i Åkersjön
+          är lätt att vandra och man kommer högt upp och kan njuta av milsvid utsikt. Önrun sägs ha ”Sveriges vidaste vy”.
+          Toppen ligger på 856 möh.</p>
+      </div>
+      <div class="head-news">
+        <h2> Nature is blooming along the Flower Trail to Ansätten</h2>
+        <img src="./nöjdfjället.jpeg" alt="nöjdfjället">
+        <p>Leden upp till fjället Önrun i Åkersjön är lätt att vandra och man kommer högt upp och kan njuta av milsvid
+          utsikt. Önrun sägs ha ”Sveriges vidaste vy”. Toppen ligger på 856 möh.Leden upp till fjället Önrun i Åkersjön
+          är lätt att vandra och man kommer högt upp och kan njuta av milsvid utsikt. Önrun sägs ha ”Sveriges vidaste vy”.
+          Toppen ligger på 856 möh.</p>
+      </div>
+    </section>
+    <!-- News boxes -->
```

Comment:
Nice and helpful to see you comment your own code, just make sure there's some space between!